src/client/pages/OfferNew/Introduction/index.tsx

```diff
@@ -50,6 +52,7 @@ const ContentContainer = styled.div`
   padding: 0 1rem;
   display: flex;
   flex-direction: column;
+  flex-direction: column-reverse;
```

Comment:
This is strange -- duplicate rule

```suggestion
  flex-direction: column-reverse;
```